### PR TITLE
Remove entry from meetingsTable when meeting ends

### PR DESCRIPTION
**Pull request by anilkumarkodali**
**Issue #:**
Unable to create a meeting with the same name with in a browser session. I either have to use a new name every time I need to run a test or restart the app to use the same name.

**Description of changes:**
Deleting the entry from the temporary cache(meetingsTable) when the meeting is ended.

**Testing:**
Tested by creating the meeting with same name with in a browser session.

*Can these tested using a demo application? Please provide reproducible step-by-step instructions.*
- Create a meeting with title "Test"
- End meeting and recreate with the same title. The meeting would be created without complaining about the meeting not being found.

**Checklist:**

1. Have you successfully run `npm run build:release` locally?
Yes

2. Do you add, modify, or delete public API definitions? If yes, has that been reviewed and approved?
No

3. Do you change the wire protocol, e.g. the request method? If yes, has that been reviewed and approved?
No

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by devalevenkatesh — APPROVED** [review]

File: demos/browser/server.js [hunk 1]
```diff
@@ -199,6 +199,8 @@ function serve(host = '127.0.0.1:8080') {
         await client.deleteMeeting({
           MeetingId: meetingTable[requestUrl.query.title].Meeting.MeetingId,
         }).promise();
+        //Remove the meeting from table(so that a new meeting can be created with the same name with in a session)
+        delete meetingTable[requestUrl.query.title];
```
Comment (ltrung) [inline demos/browser/server.js:203]: This change only apply for local dev. How about our severless demo? For that one, we currently set ttl to 24h https://github.com/aws/amazon-chime-sdk-js/blob/main/demos/serverless/src/handlers.js#L544

Comment (anilkumarkodali) [reply demos/browser/server.js:203]: I thought of updating the serverless demo too, but wasn't sure if there was a reason for setting that 24 ttl. I have the code for that as well, but did not push it. I can push that as well if you are okay with that. That way we delete the entry from dynamo if meeting ends and if all participants leave without ending the meeting, the entry would be deleted after 24 hours. Let me know.

Comment (anilkumarkodali) [reply demos/browser/server.js:203]: @ltrung - committed the handler.js code for your reference. One thing to note is there is a slight delay in UI from when the end meeting button is clicked to taking back the new meeting landing page that is introduced by the dynamo delete call. I don't think it's a big deal, but wanted to make a note of it.

**Review by ltrung — APPROVED** [review]

**Comment by ltrung** [comment]
Sorry. Forget to point out but in order to run the approval checks, you have to clone and submit your pull request. Currently, fork does not work.

**Comment by ltrung** [comment]
This actually fail the MeetingEnd test where we verified that you could not createMeeting once a meeting is deleted so closing for now.

**Pull request closed without merging**